### ISSUE TITLE
Transparent Highlight

### ISSUE THREAD

**Issue by kaiThomas236** [jgclark/vscode-todo-highlight#57]
I want to color the text of keywords but leave the backgroundcolor transparent.

**Comment by jgclark** [comment]
Is `"backgroundColor": "transparent",` not working for you?

**Comment by kaiThomas236** [comment]
yeah, that wasn't working for me. I also tried the hex code #FF000000 and the word "none". when I would try to set the bg to transparent it would just maintain the highlight color it had previously. 

unfortunately I uninstalled and made it work with another tool, so I can't get you any debug logs. if you can't duplicate it it's no big deal, maybe a one off with my system/vscode install.

**Comment by jgclark** [comment]
OK. Out of interest, after updating settings.json did you try either
toggling the highlighting off and on (using the provided command), or
switching to a different file and back again? That's often needed to get
VSCode to re-paint the file with the updating settings.
-- 
Regards, -- Jonathan

**Comment by himra13** [comment]
This works for me tho in settings.json, you just need to close the file and reopen. 
```
"backgroundColor": "transparent"
```

**Comment by jgclark** [comment]
Glad it is working for you; thanks for closing the loop on this issue, which I will now close.
